src/routes/spent/index.ts

```diff
@@ -0,0 +1,10 @@
+import ExpenseController from '../spent/controller/ExpenseController'
+import {Router} from 'express'
+
+const spentRouter = Router()
```

Comment:
Para concordar com a sugestão anterior:

```suggestion
const expenseRouter = Router()
```